v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -36,5 +36,15 @@
     "destKey": "messageId",
     "sourceKeys": "messageId",
     "required": false
+  },
+  {
+    "destKey": "properties.$group_set",
+    "sourceKeys": "traits.groupProperties",
+    "required": false
+  },
```

Comment:
We can map the entire traits as group_set. Because the traits are basically group properties.

Reply:
Updated.